ethcore/vm/src/ext.rs

```diff
@@ -166,6 +166,9 @@ pub trait Ext {
 	/// Prepare to trace an operation. Passthrough for the VM trace.
 	fn trace_prepare_execute(&mut self, _pc: usize, _instruction: u8, _gas_cost: U256, _mem_written: Option<(usize, usize)>, _store_written: Option<(U256, U256)>) {}
```

Comment:
Can we improve the docs here and underline the fact that this call is ALWAYS followed by either `trace_executed` or `trace_failed`. and that the implementation is responsible to clean up in either/both of those methods.